elisapolystar/zeebe-test-bench
# vnoc-8591 create test bench

### Description
- A Docker environment with Zeebe (with embedded gateway), zeebe-kafka-exporter, Kafka, Postgres, and Zeebe Simple Monitor.
- Some process files in BPMN format ready to be deployed to Zeebe
- Some pre-defined job workers to be called from BPMN files.
- A detailed README.

### Testing
- Start the docker environment: `./start.sh`.
- Start job worker: Run job workers in IntelliJ
- Process and process instance can be deployed and created using `zbctl`. Instruction on using it is available in README.
- Job worker and message publisher can also be handled using `zbctl`. More details in README.
- Status of the instance can be observed in Zeebe Simple Monitor.
- The README should be detailed and easy to follow, even if one has little knowledge in development. University students will be the targeted audience of this repo.
